File: getCount.cpp
#include <string>
#include <algorithm>
#include <cctype>

using namespace std;

int getCount(const string& inputStr){
  int num_vowels = 0;
  string word = inputStr;
  transform(word.begin(), word.end(), word.begin(), ::tolower);
  vector<char> vowels = {'a', 'e', 'i', 'o', 'u'};
  
  for (char letter : word) {
    if (find(vowels.begin(), vowels.end(), letter) != vowels.end()) {
      num_vowels++;
    }
  }
  
  return num_vowels;
}

File: getCount.java
import java.util.Arrays;

public class Vowels {

  public static int getCount(String str) {
    int sum = 0;
    char[] vowels = {'a', 'e', 'i', 'o', 'u'};
    
    for (char letter : str.toLowerCase().toCharArray()) {
      if (Arrays.binarySearch(vowels, letter) >= 0) {
        sum++;
      }
    }
    
    return sum;
  }
}
